tests/test_applications.py

```diff
@@ -121,8 +115,13 @@ def custom_ws_exception_handler(websocket: WebSocket, exc: CustomWSException):
         Mount("/users", app=users),
         Host("{subdomain}.example.org", app=subdomain),
     ],
-    exception_handlers=exception_handlers,
     middleware=middleware,
+    exception_handlers={
+        500: error_500,
+        405: method_not_allowed,
+        HTTPException: http_exception,
+        CustomWSException: custom_ws_exception_handler,
+    },
```

Comment:
Why is this needed?

Reply:
I don't think that makes sense. How did you prove this assumption?

Reply:
by running `scripts/lint`.  

My guess is the following:  MyPy will check that the type of a _literal_ matches the argument list, but it will not automatically and implicitly assign a type to a variable (`exception_handers`) based on the literal used to initialize it.  Since the variable has no type, it won't be checked against the arbument list, and mypy isn't in --strict mode and doesn't complain about it.

Anyway, you can clearly see that the type of `custom_ws_exception_handler`, does not match the hints, yet mypy does not complain about this dict.

Reply:
The problem was that some of the exception handlers were untyped (ANY) and then the whole list collapses to that and even your carefully typed exception handlers don't get any testing.